test/rekt/features/channel/data_plane.go

```diff
@@ -56,16 +61,24 @@ func DataPlaneChannel(channelName string) *feature.Feature {
 	f.Requirement("Channel is Ready", channel_impl.IsReady(channelName))
 
 	f.Stable("Input").
-		Must("Every Channel MUST expose either an HTTP or HTTPS endpoint.", todo).
+		Must("Every Channel MUST expose either an HTTP or HTTPS endpoint.", func(ctx context.Context, t feature.T) {
+			c := getChannelable(ctx, t)
+			addr := *c.Status.AddressStatus.Address
+			checkScheme(t, addr)
+		}).
```

Comment:
@sadath-12 instead of having this function as an anonymous function, can you create it as a named function? The body of the function looks correct, but it should be a named function like all the other functions